tests/e2e/test_rosacli_account_roles.go

```diff
@@ -829,6 +833,124 @@ var _ = Describe("Create account roles", labels.Feature.AccountRoles, func() {
 		ocmResourceService = rosaClient.OCMResource
 	})
 
+	AfterEach(func() {
+		By("Cleanup account-roles")
+		if len(accountRolePrefixesNeedCleanup) > 0 {
+			for _, v := range accountRolePrefixesNeedCleanup {
+				_, err := ocmResourceService.DeleteAccountRole("--mode", "auto",
+					"--prefix", v,
+					"-y")
+
+				Expect(err).To(BeNil())
+			}
+		}
+	})
+	It("to check the trust policy attach information when create account-roles multiple times - [id:75904]",
+		labels.High, labels.Runtime.OCMResources,
+		func() {
+			accoutRoleNames := []string{
+				"ControlPlane-Role",
+				"Worker-Role",
+				"Support-Role",
+				"Installer-Role",
+				"HCP-ROSA-Installer-Role",
+				"HCP-ROSA-Support-Role",
+				"HCP-ROSA-Worker-Role",
+			}
+			awsAccounts := []string{
```

Comment:
I guess billing account number is a sensitive information, instead of direct use can we make a constant and use here?

Reply:
awsAccounts are jump account, not the billing account. It is visible to customers. I will add the variable 'JumpAccounts' in common/constants/general.go anyway, no functional difference.

Reply:
Thanks Yu, some new information to me :-)